app/javascript/components/recipes/new/create-recipes.vue

```diff
@@ -70,31 +71,92 @@
         @click="create"
       />
     </div>
+
+    <base-modal
+      @ok="addIngredientstoRecipe"
+      @cancel="toggleAddIngredientModal"
+      v-if="showingAddIngredientModal"
+      :title="$t('msg.ingredients.add')"
+      :ok-button-label="$t('msg.add')"
+      :cancel-button-label="$t('msg.cancel')"
+    >
+      <div class="flex f-row w-full">
+        <div class="mr-10">
+          <search-ingredient-list 
+            :ingredients="getIngredients"
+            @add-ingredient="addIngredient"
+          />
+        </div>
+        <selected-ingredients 
+          :elements="selectedIngredients"
+          :svg="{sixdots: true, cancel: false, menu_recipe: true}"
+          :input="false"
+          :placeholder="'-'"
+          @delete-ingredient="deleteIngredient"
+        /> 
+      </div>
+    </base-modal>
+
   </div>
 </template>
 
 <script>
 import List from './list.vue';
+import BaseModalComponent from '../../base/base-modal.vue';
+import SelectedIngredients from './selected-ingredients.vue';
+import SearchIngredientList from './search-ingredient-list.vue';
 
 export default {
   components: {
     List,
+    BaseModalComponent,
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>  The "BaseModalComponent" component has been registered but not used           vue/no-unused-components